src/server_manager/web_app/outline-gcp-create-server-app.ts

```diff
@@ -20,8 +20,9 @@ import '@polymer/paper-item/paper-item';
 import './ui_components/outline-step-view';
 import './ui_components/outline-region-picker-step';
 
-import {css, customElement, html, internalProperty, LitElement, property} from 'lit-element';
-import {unsafeHTML} from 'lit-html/directives/unsafe-html';
+import {css, html, LitElement} from 'lit';
```

Comment:
I see a number of compilation issues: https://github.com/Jigsaw-Code/outline-server/actions/runs/7646972769/job/20836953270?pr=1483

Please fix.